### PR TITLE
Rewrite Morphic section about Interactors, now called Dialog Windows

### PR DESCRIPTION
**Pull request by LinqLover**
Introduces the concepts Project and UIManager. Figures were updates accordingly.

See <https://github.com/codeZeilen/SqueakByExample-english/issues/21#issuecomment-695981273-permalink:~:text=yes.-,%40codeZeilen%20%40tom95%20Chapter%2011.6%20(Interactions)%20describes%20the%20use%20of%20FillInTheBlank%20and%20PopUpMenu.%20Shouldn't%20these%20classes%20be%20rather%20considered%20as%20an%20implementation%20detail%20of%20UIManager%20but%20not%20be%20addressed%20directly%3F%20Would%20it%20be%20better%20to%20replace%20the%20code%20example,%F0%9F%91%8D%201>.

### CODE REVIEW

File: Morphic/Morphic.tex [hunk 1]
```diff
@@ -593,69 +593,57 @@ \subsection{Morphic animations}
...
-%	\caption{Dialog displayed by \ct{FillInTheBlank request: 'What''s your name?' initialAnswer: 'no name'}.
-%		\label{fig:dialogName}}
-%\end{figure}
+To prompt the user for input, the \clsind{UIManager} class provides a few ready-to-use dialog boxes.
+For instance, the \mthind{UIManager}{request:initialAnswer:} method returns the string entered by the user (\figref{dialogName}).
+However, \clsind{UIManager} is an abstract class that has several subclasses such as \clsind{MorphicUIManager}, \clsind{MVCUIManager}, and others.
+What's the reason for this?
+
+\sq supports a wide range of different UI systems of which Morphic is only one of many.
+For instance, another UI system in \sq is \emph{MVC} (named after its tripartite Model View Controller architecture) which was the only available UI system in classic Smalltalk-80.
```
Comment (codeZeilen) [inline Morphic/Morphic.tex:635]: Model-View-Controller architecture

File: Morphic/Morphic.tex [hunk 2]
```diff
@@ -593,69 +593,57 @@ \subsection{Morphic animations}
...
+To allow developers to write dialogs that work for any UI system, \sq provides the \clsind{UIManager} facade that defines a consistent interface to invoke user dialogs for any UI system.
+Every UI system then is represented by a subinstance of the class \clsind{Project} which instantiates a customized subclass of the \clsind{UIManager}.
```
Comment (codeZeilen) [inline Morphic/Morphic.tex:637]: remove facade, to specific for an introductory book

Comment (codeZeilen) [inline Morphic/Morphic.tex:637]: To get the \clsind{UIManager} for our UI system, we can ask the class \clsind{Project}, which represents a running UI system. The class \clsind{Project} takes care of instantiating the \clsind{UIManager} corresponding to our UI system.

File: Morphic/Morphic.tex (hunk 1 above)
Comment (LinqLover) [reply Morphic/Morphic.tex]: Resolved via 719dc16

File: Morphic/Morphic.tex (hunk 2 above)
Comment (LinqLover) [reply Morphic/Morphic.tex]: 719dc16

Comment (LinqLover) [reply Morphic/Morphic.tex]: Please check 719dc16

**Review by codeZeilen — APPROVED** [review]

**Pull request merged**